### PR TITLE
NR-150571: fix 'inSeconds' metrics unit

### PR DESCRIPTION
**Pull request by sigilioso**
fixes: #111

### CODE REVIEW

File: src/collection_test.go [hunk 1]
```diff
@@ -180,8 +190,19 @@ func Test_collectMetrics(t *testing.T) {
...
+	assert.Equal(t, float64(0.011), backendEntity.Metrics[0].Metrics["backend.averageConnectTimeInSeconds"])
+	assert.Equal(t, float64(0.012), backendEntity.Metrics[0].Metrics["backend.averageResponseTimeInSeconds"])
+	assert.Equal(t, float64(0.013), backendEntity.Metrics[0].Metrics["backend.averageTotalSessionTimeInSeconds"])
+
 	assert.Equal(t, float64(1.0), serverEntity.Metrics[0].Metrics["server.currentSessions"])
+	assert.Equal(t, float64(0.1), serverEntity.Metrics[0].Metrics["server.averageQueueTimeInSeconds"])
+	assert.Equal(t, float64(0.101), serverEntity.Metrics[0].Metrics["server.averageConnectTimeInSeconds"])
+	assert.Equal(t, float64(0.102), serverEntity.Metrics[0].Metrics["server.averageResponseTimeInSeconds"])
+	assert.Equal(t, float64(0.103), serverEntity.Metrics[0].Metrics["server.averageTotalSessionTimeInSeconds"])
+	assert.Equal(t, float64(0.104), serverEntity.Metrics[0].Metrics["server.agentDurationSeconds"])
```
Comment (marcsanmi) [inline src/collection_test.go:205]: Is it `agentDurationSeconds` or `agentDurationInSeconds`?

Comment (sigilioso) [reply src/collection_test.go:205]: It is `agentDurationSecons` (names are not changed, we decided to fix the values but keep the names as they were). No idea why this the only one without `In`.

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 Unreleased section should follow [Release Toolkit](https://github.com/newrelic/release-toolkit#render-markdown-and-update-markdown)
 ## Unreleased
+### breaking
+- `*InSeconds` metrics are now reported in seconds. The affected metrics used to be reported in milliseconds.
+  Affected metrics: `backend.averageQueueTimeInSeconds`, `backend.averageResponseTimeInSeconds`,
+  `backend.averageTotalSessionTimeInSeconds`, `server.agentDurationInSeconds`, `server.averageConnectTimeInSeconds`,
```
Comment (marcsanmi) [inline CHANGELOG.md:13]:
```suggestion
  `backend.averageTotalSessionTimeInSeconds`, `server.agentDurationSeconds`, `server.averageConnectTimeInSeconds`,
```

Comment (sigilioso) [reply CHANGELOG.md:13]: 🎯 🙇

**Review by marcsanmi — APPROVED** [review]
LGTM 👍

**Pull request merged**